write-question.c

```diff
@@ -0,0 +1,80 @@
+// C program to Open a File, 
+// Write in it, And Close the File 
+
+# include <stdio.h> 
+#include<stdlib.h>
+# include <string.h>
+           
+    struct test_case{
+       char input [1000];
```

Comment:
Declare all the structures in "coff.h" as these structures will be used in rest of the programs too.

Also the size constants like "1000".

```
#define coff_test_case_input_size 1000
struct test_case{
  char input[coff_test_case_input_size];
```

Reply:
Also use "spaces" instead of "tab"
And there should be only two space gap for nested block.
For ex:
Don't do this
```
struct xyz {
    char abc;
```

Use two spaced, no tab;
```
struct xyz {
  char abc;
```